assets/components/tinymcerte/js/vendor/tinymce/plugins/modxlink/plugin.min.js

```diff
@@ -120,6 +120,49 @@ tinymce.PluginManager.add('modxlink', function(editor) {
 			if (!meta.text) {
 				updateText.call(this);
 			}
+
+			getPagetitleForModxLink(this.value());
+		}
+
+		function isModxLink(link)
+		{
+			return (link.startsWith('[[~') && link.endsWith(']]'));
+		}
+
+		function getPagetitleForModxLink(link)
+		{
+			if (!isModxLink(link)) {
+				setRenderedPagetitle('');
+			}
```

Comment:
The next part should run only, when link contains a MODX link tag, so an else should be used.